app/lib/router/router.dart

```diff
@@ -876,119 +871,106 @@ List<RouteBase> makeRoutes(Ref ref) {
         ),
 
         /// Space subshell
-        ShellRoute(
-          navigatorKey: spaceNavKey,
-          pageBuilder: (context, state, child) {
+        GoRoute(
+          parentNavigatorKey: shellNavKey,
```

Comment:
do these really need to the `parentNavigatorKey: shellNavKey,`?

Reply:
On desktop view, when I open space from chat parent space badge icon, space was opening on side view. So make it on full view i have added this key.